CONTRIBUTING.md

```diff
@@ -8,4 +8,4 @@ These rules ensure that the code will remain consistent and the project is maint
 - You must follow our coding standards (You can see the rulesets in the [coding-standards](https://github.com/shopsys/coding-standards) repository).
 - The build of your branch on [Travis CI](https://travis-ci.org/shopsys/phpstorm-inspect/branches) must be passing (your branch will be automatically built after push).
 - It is necessary to put information about your code being provided under MIT license in the description of your pull request.
-- A form of your commits should meet our [guidelines](https://git.shopsys-framework.com/shopsys/shopsys-framework/blob/master/docs/contributing/guidelines-for-creating-commits.md).
\ No newline at end of file
+- A form of your commits should meet our [guidelines](https://github.com/shopsys/shopsys/blob/master/docs/contributing/guidelines-for-creating-commits.md).
```

Comment:
Hey Svaťa, your links does not contain latest divorce of FE a BE, and docs are now situated in project-base folder. If you try the link you written here, you will see it is not working.